core/src/main/scala/org/bitcoins/core/api/wallet/WalletApi.scala

```diff
@@ -364,6 +363,10 @@ trait WalletApi extends StartStopAsync[WalletApi] {
       amounts: Vector[CurrencyUnit],
       feeRate: FeeUnit)(implicit ec: ExecutionContext): Future[Transaction]
 
+  def bumpFee(
```

Comment:
Scaladoc on what method this scala method is using to bump the fee? Is it using CPFP or RBF? I would be supportive of changing the method name to 

`bumpFeeCFPF()` or `bumpFeeRBF()`. 

Or we can make it even more complex by adding an extra parameter to the method that with an ADT like this

```scala

sealed trait FeeBumpMethod
object FeeBumpMethod { 
  final case object CPFP extends FeeBumpMethod
  final case object RBF extends FeeBumpMethod
}
```

and then just add that as a parameter. This might be nice if there are future fee bump methods -- but it is more work.

Reply:
Changed the name of the function to `bumpFeeRBF`. In the future we can make `bumpFeeCPFP`